pkg/cmd/bootstrap.go

```diff
@@ -146,10 +148,10 @@ func missingFlagErr(flags []string) error {
 }
 
 // initiateInteractiveMode starts the interactive mode impplementation if no flags are passed.
-func initiateInteractiveMode(io *BootstrapParameters) error {
+func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client) error {
 	log.Progressf("\nStarting interactive prompt\n")
 	// ask for sealed secrets only when default is absent
-	if io.SealedSecretsService == (types.NamespacedName{}) {
+	if client.CheckIfSealedSecretsExists(sealedSecretDefaultValue) != nil {
 		io.SealedSecretsService.Name = ui.EnterSealedSecretService(&io.SealedSecretsService)
 	}
 	io.GitOpsRepoURL = utility.AddGitSuffixIfNecessary(ui.EnterGitRepo())
```

Comment:
client.CheckIfSealedSecretsExists(sealedSecretDefaultValue) can return some error other than "Not found".

Also, what if the user's entered SS service does not exist?

Reply:
Yes William thanks for bringing that up, the prompt now throws errors based on whether the namespace was missing or the service is missing. Thank you.

Reply:
@gaganhegde I'm not sure this is the right fix.

The issue before was that if it got an error checking for sealed secrets, it assumed it was because the service couldn't be found, so prompted for the name/namespace.

What happens if it can't connect to the Kubernetes server? What happens if it gets an auth failure?

Is prompting for the name/namespace the right thing to do in that case?